TestingApp/ViewController/ViewController+CallVisualizer.swift

```diff
@@ -12,12 +12,12 @@ extension ViewController {
     }
     
     func showVisitorCode(presentation: CallVisualizer.Presentation) {
-        showRemoteConfigAlert { [weak self] fileName in
-            let config = self?.retrieveRemoteConfiguration(fileName)
+//        showRemoteConfigAlert { [weak self] fileName in
```

Comment:
JFYI. I made applying remote config optional in this PR but haven't merged yet.
https://github.com/salemove/ios-sdk-widgets/pull/489

Reply:
I've merged it to master